website/content/api-docs/secret/key-management/gcpkms.mdx

```diff
@@ -0,0 +1,47 @@
+---
+layout: api
+page_title: GCP Cloud KMS - Key Management - Secrets Engines - HTTP API
+description: The GCP Cloud KMS API documentation for the Key Management secrets engine.
+---
+
+# GCP Cloud KMS (API)
+
+The Key Management secrets engine supports lifecycle management of keys in [GCP Cloud KMS](https://cloud.google.com/security-key-management)
+[key rings](https://cloud.google.com/kms/docs/resource-hierarchy#key_rings). This is accomplished by
```

Comment:
These links are very close together and hard to tell its two links when you see it on the website. Wondering if we can put a filler word in between them?

Reply:
Good idea. Removed one of the links in 1533b8b.